.github/workflows/tests.yml

```diff
@@ -2,6 +2,15 @@ name: tests
 on: [pull_request]
 jobs:
 
+  add-label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: buildsville/add-remove-label@v1
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        label: work-in-progress
+        tyle: add
```

Comment:
@anirudhprabhakaran3 Shouldn't tyle be type?

Reply:
Yeah, I just fixed that in the newest commit. Thanks for pointing it out!